packages/fiori/src/NavigationLayout.ts

```diff
@@ -0,0 +1,106 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import browserScrollbarCSS from "@ui5/webcomponents/dist/generated/themes/BrowserScrollbar.css.js";
+import {
+	isPhone,
+	isTablet,
+	isCombi,
+} from "@ui5/webcomponents-base/dist/Device.js";
+import type SideNavigation from "./SideNavigation.js";
+
+// Template
+import NavigationLayoutTemplate from "./generated/templates/NavigationLayoutTemplate.lit.js";
+
+// Styles
+import NavigationLayoutCss from "./generated/themes/NavigationLayout.css.js";
+
+/**
+ * @class
+ *
+ * ### Overview
+ *
+ * The `ui5-tool-page` is a container ...
+ * ### ES6 Module Import
+ *
+ * `import "@ui5/webcomponents-fiori/dist/NavigationLayout.js";`
+ * @constructor
+ * @extends UI5Element
+ * @since 2.1.0
+ * @public
+ */
+@customElement({
+	tag: "ui5-navigation-layout",
+	languageAware: true,
+	renderer: litRender,
+	styles: [
+		browserScrollbarCSS,
+		NavigationLayoutCss,
+	],
+	template: NavigationLayoutTemplate,
+})
+class NavigationLayout extends UI5Element {
+	_sideCollapsed = isPhone() || (isTablet() && !isCombi());
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	isPhone = isPhone();
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	isTablet = isTablet() && !isCombi();
+
+	/**
+	 * Indicates whether if the side menu is collapsed.
+	 * @public
+	 */
+	@property({ type: Boolean })
+	set sideCollapsed(value: boolean) {
```

Comment:
doesn't work if `<ui5-navigation-layout side-collapsed>` or on tablet. Side is expanded and first click keeps it expanded.